script/Systems/Overlay/HUD.lua

```diff
@@ -1637,28 +1486,37 @@ function HUD:drawPlayerHullInteg(a)
     -- end
 
     -- Draw text of player ship name
-    HUD:drawHudTextDouble(164, sy - 320, Config.ui.color.meterBarLight, hudFontSize, 0.5, playerShip:getName())
+    HUD:drawHudTextDouble(164, sy - 280, Config.ui.color.meterBarLight, hudFontSize, 0.5, playerShip:getName())
 
     -- Draw hologram of player ship on a grid background
-    UI.DrawEx.Grid(114, sy - 231, 100, 55, Config.ui.color.meterBarLight)
-    UI.DrawEx.Hologram(playerShip.mesh, 34, sy - 336, 260, 260, Config.ui.color.hologram, playerZoom / 1.7, -1.57, 0.0)
-
-    -- Draw player ship data as meters
-    UI.DrawEx.RectOutline(66, sy - 96, 202, 22, Config.ui.color.borderBright)
-    UI.DrawEx.Meter(72, sy - 90, 10, 10, Config.ui.color.shieldStrength, 10, 10, floor(playerShieldPct / 10), false, nil,
-        1)
-    UI.DrawEx.RectOutline(66, sy - 66, 202, 22, Config.ui.color.borderBright)
-    UI.DrawEx.Meter(72, sy - 60, 10, 10, Config.ui.color.armorIntegrity, 10, 10, floor(playerArmorPct / 10), false, nil,
-        1)
-    UI.DrawEx.RectOutline(66, sy - 36, 202, 22, Config.ui.color.borderBright)
-    UI.DrawEx.Meter(72, sy - 30, 10, 10, hc, 10, 10, floor(playerHealthPct / 10), false, nil, 1)
-
-    -- OBSOLETE?: Also draw the player ship's health bar under the central reticle
-    -- UI.DrawEx.RectOutline(cx - 22, cy + 18, 44, 8, Config.ui.color.borderDim)
-    -- UI.DrawEx.Rect(cx - 20, cy + 20, 40, 4, Config.ui.color.healthColor[playerHealthCI])
+--    UI.DrawEx.Grid(114, sy - 180, 100, 100, Config.ui.color.meterBarLight)
+    UI.DrawEx.Hologram(playerShip.mesh, 34, sy - 260, 260, 260, Config.ui.color.hologram, playerZoom / 1.7, -1.57, 0.0)
+
+    -- Draw the current percentage statuses of key player ship systems as concentric rings
+    for str = floor(playerShieldPct / 10), 1, -1 do
+        UI.DrawEx.RingDim(164, sy - 130, 110 + (str - 1), Config.ui.color.shieldStrength)
+    end
+    for str = floor(playerArmorPct / 10), 1, -1 do
+        UI.DrawEx.RingDim(164, sy - 130,  95 + (str - 1), Config.ui.color.armorIntegrity)
+    end
+    for str = floor(playerHealthPct / 10), 1, -1 do
+        UI.DrawEx.RingDim(164, sy - 130,  80 + (str - 1), Config.ui.color.hullIntegrity)
+    end
+    for str = floor(playerCapChgPct / 10), 1, -1 do
+        UI.DrawEx.RingDim(164, sy - 130,  65 + (str - 1), Config.ui.color.capacitorEnergy)
+    end
+
+    -- Also draw player ship hull, armor, and shield strengths in text form
+    local text = ""
+    text = format("%d%%", playerHealthPct)
+    HUD:drawHudTextDouble(164, sy - 54, Config.ui.color.meterBarLight, hudFontSize - 2, 0.5, text)
+    text = format("%d%%", playerArmorPct)
+    HUD:drawHudTextDouble(164, sy - 38, Config.ui.color.meterBarLight, hudFontSize - 2, 0.5, text)
+    text = format("%d%%", playerShieldPct)
+    HUD:drawHudTextDouble(164, sy - 22, Config.ui.color.meterBarLight, hudFontSize - 2, 0.5, text)
 end
 
-function HUD:drawTargetHullInteg(a)
+function HUD:drawTargetSystemInteg(a)
```

Comment:
Can you also change this to use a full description name?

Reply:
Done.